bzzeth/bzzeth.go

```diff
@@ -135,8 +143,7 @@ func (b *BzzEth) handleNewBlockHeaders(ctx context.Context, p *Peer, msg *NewBlo
 	}
 	yes, err := b.netStore.Store.HasMulti(ctx, addresses...)
 	if err != nil {
-		log.Error("Error checking hashesh in store", "Reason", err)
-		return nil
+		return fmt.Errorf("error checking hashesh in store. Reason: %w", err)
```

Comment:
I do not think that `error` and `Reason` is needed in message, Maybe something like `check hashes: %w`?